apps/web/src/ee/clerk/components/UserProfileButton.tsx

```diff
@@ -0,0 +1,27 @@
+import { UserButton, UserProfile } from '@clerk/clerk-react';
+import { IconCreditCard, IconGroup, IconRoomPreferences, IconWorkspacePremium } from '@novu/design-system';
+import { ROUTES } from '../../../constants/routes';
+import { BrandingPage } from '../../../pages/brand/BrandingPage';
+import { BillingPage } from '../../billing/pages/BillingPage';
+import { CustomOrganizationProfile } from './CustomOrganizationProfile';
+
+export function UserProfileButton() {
+  return (
+    <UserButton afterSignOutUrl={ROUTES.AUTH_LOGIN}>
+      <UserButton.UserProfilePage label="account" />
+      <UserButton.UserProfilePage label="Organisation" url={ROUTES.ORGANIZATION} labelIcon={<IconRoomPreferences />}>
+        <CustomOrganizationProfile firstItem="general" />
+      </UserButton.UserProfilePage>
+      <UserButton.UserProfilePage label="security" />
```

Comment:
```suggestion
      <UserButton.UserProfilePage label="Security" />
```

Reply:
this one unfortunately cannot be changed (or I haven't yet found a way how)

```typescript
type UserProfilePageProps = PageProps<'account' | 'security'>;
```